Swinject/Container+SwinjectStoryboard.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Container+SwinjectStoryboard.swift
+//  Swinject
+//
+//  Created by Yoichi Tagaya on 11/28/15.
+//  Copyright © 2015 Swinject Contributors. All rights reserved.
+//
+
+#if os(iOS) || os(OSX) || os(tvOS)
+extension Container {
+    /// Adds a registration of the specified view or window controller that is configured in a storyboard.
+    ///
+    /// - Note: Do NOT explicitly resolve the controller registered by this method.
+    ///         The controller is intended to be resolved by `SwinjectStoryboard` implicitly.
+    ///
+    /// - Parameters:
+    ///   - controllerType: The controller type to register as a service type.
+    ///                     The type is `UIViewController` in iOS, `NSViewController` or `NSWindowController` in OS X.
+    ///   - name:           A registration name, which is used to differenciate from other registrations
+    ///                     that have the same view or window controller type.
+    ///   - initCompleted:  A closure to specifiy how the dependencies of the view or window controller are injected.
+    ///                     It is invoked by the `Container` when the view or window controller is instantiated by `SwinjectStoryboard`.
```

Comment:
Length Violation: Line should be 100 characters or less: currently 136 characters